core/federated/RTI/rti.c

```diff
@@ -79,9 +89,20 @@ int main(int argc, const char* argv[]) {
     }
     int socket_descriptor = start_rti_server(_RTI.user_specified_port);
     wait_for_federates(socket_descriptor);
+    
+    termination();
+
+    return 0;
+}
+
+/**
+ * @brief A clean termination of the RTI will write the trace file, if tracing is
+ * enabled, before exiting.
+ */
+void termination() {
     if (_RTI.tracing_enabled) {
         stop_trace();
-    }
+        printf("RTI trace file saved.\n");
```

Comment:
I think we  should use the `lf_print` functions rather than `printf`, because there are platforms which doesn't support `printf` but still can provide some implementation of `lf_print`.

Reply:
Indeed!
The RTI was not using the abstraction provided by LF, as pointed out by @siljesu.
Will update them.